<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serial</class>
 <widget class="QMainWindow" name="Serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="baudrateBox">
    <property name="geometry">
     <rect>
      <x>748</x>
      <y>89</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="DatabitBox">
    <property name="geometry">
     <rect>
      <x>748</x>
      <y>143</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="parityBox">
    <property name="geometry">
     <rect>
      <x>748</x>
      <y>197</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="StopbitBox">
    <property name="geometry">
     <rect>
      <x>748</x>
      <y>251</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comBox">
    <property name="geometry">
     <rect>
      <x>748</x>
      <y>35</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="BaudRatelabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Baud Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="DataBitlabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  DataBit</string>
    </property>
   </widget>
   <widget class="QLabel" name="Paritylabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>190</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>  Parity</string>
    </property>
   </widget>
   <widget class="QLabel" name="StopBitlable">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> Stop Bits</string>
    </property>
   </widget>
   <widget class="QLabel" name="comlable">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>    Com</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="replyConsole">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>611</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="statusLine">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>560</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ComSwitch">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>500</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>20</width>
      <height>471</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>470</y>
      <width>891</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>611</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OscButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>410</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Oscilloscope</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHRT_D">
    <property name="title">
     <string>HRT_D</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuContract">
    <property name="title">
     <string>Contract</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQi_shield95_gmail_com"/>
    <addaction name="action2448613659"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionClick_me"/>
   </widget>
   <widget class="QMenu" name="menuTheme">
    <property name="title">
     <string>Theme</string>
    </property>
   </widget>
   <addaction name="menuHRT_D"/>
   <addaction name="menuContract"/>
   <addaction name="menuHelp"/>
   <addaction name="menuTheme"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQi_shield95_gmail_com">
   <property name="text">
    <string>qi.shield95@gmail.com</string>
   </property>
  </action>
  <action name="action2448613659">
   <property name="text">
    <string>2448613659</string>
   </property>
  </action>
  <action name="actionClick_me">
   <property name="text">
    <string>Click me</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
